app/models/user.rb

```diff
@@ -106,29 +106,29 @@ class User < ApplicationRecord
             format: /\A(http(s)?:\/\/)?(www.twitch.tv|twitch.tv)\/.*\Z/
   validates :shirt_gender,
             inclusion: { in: %w[unisex womens],
-                         message: "%{value} is not a valid shirt style" },
+                         message: "%<value>s is not a valid shirt style" },
```

Comment:
I have never seen this syntax before, do you have a link to the docs for it? I was coming up empty-handed trying to google.

Reply:
I got it from [rubocop's Style/FormatStringToken](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/FormatStringToken) and from the [Ruby Style Guide's 
Named Format Tokens rule](https://github.com/rubocop-hq/ruby-style-guide#named-format-tokens).

I think they both refer to [Kernel#format](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-format)'s advanced examples:

```text
For more complex formatting, Ruby supports a reference by name. %<name>s style uses format style, but %{name} style doesn't.

Examples:

sprintf("%<foo>d : %<bar>f", { :foo => 1, :bar => 2 })
  #=> 1 : 2.000000
sprintf("%{foo}f", { :foo => 1 })
  # => "1f"
```

Basically the reason is to embed the value type in the format string

Reply:
TIL, thanks!